tools/ds-identify

```diff
@@ -1739,6 +1739,9 @@ read_config() {
     for tok in ${DI_KERNEL_CMDLINE}; do
         key=${tok%%=*}
         val=${tok#*=}
+
+        # discard anything after delimiter
+        val=$(echo $val | cut -f1 -d ';')
```

Comment:
I think we probably want to avoid subshell and cut because we can use POSIX variable substitution for this (like we are doing a couple lines above).

let's try this instead
```suggestion
        val=${val%;*}
```
	
tested locally:
```
$ sh -c 'a="nocloud-net;s=http"; echo ${a%;*}'
nocloud-net
```